### ISSUE TITLE
Unable to manage tiles citizens are working

### ISSUE THREAD

**Issue by Deddryk** [Azurency/CQUI_Community-Edition#354]
When attempting to manage the tiles citizens are working, the UI does not populate the expected buttons to select tiles to work.

**To Reproduce**
1. Select a city
3. No buttons appear to change the tiles citizens are working

**Expected behavior**
Buttons should appear that allow me to change the tiles citizens are working.

**Screenshots**
![image](https://user-images.githubusercontent.com/2798989/43873879-94d7fa6c-9b56-11e8-9dfc-5691ac6e7292.png)

 - OS: Ubuntu 16.04
 - Version 1.0.0.220
 - Expansion Rise and Fall
 - Other mods [e.g. YnAMP]

**Additional context**
I'm not sure if this mod is intended to be backwards compatible and I know that the current version is compatible with 1.0.0.226 so feel free to close if the issue is just that the newest patch is not yet available for linux.

**Comment by Azurency** [comment]
Oh, so linux is still not updated with the Spring patch ? still in 1.0.0.220 ? If that's the case yes it's not compatible, you might want to try an earlier version available here https://github.com/Azurency/CQUI_Community-Edition/releases/tag/mac-1.0.0.220

**Comment by Deddryk** [comment]
Yeah, we linux plebs are still waiting on the spring update.  The old version works great in vanilla but a few things are buggy with Rise and Fall.  Thanks for the response.  Guess I'll just have to wait.
